ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -113,6 +113,7 @@
         .paymentAmount("400000")
         .paymentReference("RC-1524-6488-1670-7520")
         .claimSubmissionOperationIndicators(getDefaultCCDClaimSubmissionOperationIndicators.get())
+        .state("create")
```

Comment:
Prefer to use `ClaimState.CREATE.getValue()` instead of string here

Reply:
done